docs/expectation_gallery/azure-pipelines-expectation-gallery.yml

```diff
@@ -162,76 +539,230 @@ stages:
+            displayName: 'Show full checklist summary'
+
+          - bash: grep -E "(^expect|Completeness checklist|^ *[A-z]|^ *-|-----)" checklists--snowflake.txt | grep -vE '(No validate_configuration|Using default validate_configuration|Has a full suite|Has passed a manual)'
+            workingDirectory: $(Build.SourcesDirectory)/assets/scripts/
+            displayName: 'Show checklist issues'
+
+          - bash: cat testing-error-messages--snowflake.txt
+            workingDirectory: $(Build.SourcesDirectory)/assets/scripts/
+            displayName: 'Show testing errors'
+
+          - bash: grep --color -n -i warning -B 2 output--build_gallery--snowflake.txt || echo "No warnings found"
+            workingDirectory: $(Build.SourcesDirectory)/assets/scripts/
+            displayName: 'Show testing warnings'
+
+          - bash: cat gallery-tracebacks--snowflake.txt
+            workingDirectory: $(Build.SourcesDirectory)/assets/scripts/
+            displayName: 'Show gallery tracebacks'
+
+          - bash: grep "to df.to_sql" testing-times--snowflake.txt || echo "No df.to_sql calls were made"
+            workingDirectory: $(Build.SourcesDirectory)/assets/scripts/
+            displayName: 'Show DataFrame to SQL times'
+
+          - bash: grep "to run" testing-times--snowflake.txt
+            workingDirectory: $(Build.SourcesDirectory)/assets/scripts/
+            displayName: 'Show testing times grouped by backend and Expectation'
+
+          - bash: grep "to evaluate_json" testing-times--snowflake.txt
+            workingDirectory: $(Build.SourcesDirectory)/assets/scripts/
+            displayName: 'Show testing times for individual tests'
+
+          - task: S3Upload@1
+            inputs:
+              regionName: 'us-east-2'
+              awsCredentials: 'aws-ci-great-expectations'
+              bucketName: 'superconductive-public'
+              sourceFolder: '$(Build.SourcesDirectory)/assets/scripts'
+              globExpressions: '*.json'
+              targetFolder: 'static/gallery/'
+              filesAcl: 'public-read'
+
+
+      - job: aws
+        timeoutInMinutes: 240
+        variables:
+          python.version: '3.8'
+          GE_pytest_opts: ''
+
+        steps:
+          - task: UsePythonVersion@0
+            inputs:
+              versionSpec: '$(python.version)'
+            displayName: 'Use Python $(python.version)'
+
+          - bash: python -m pip install --upgrade pip
+            displayName: 'Update pip'
+
+          - script: |
+              pip install \
+                "cryptography==38.0.4" \
+                --requirement reqs/requirements-dev-all-contrib-expectations.txt \
+                --editable ".[spark, redshift, athena]" \
+                --constraint constraints-dev.txt
+            displayName: 'Install dependencies'
+
+            #
+            # Update to `--backends "redshift, athena"` when athena testing is real
```

Comment:
How about we make a separate redshift and athena job instead of trying to merge them into aws?